climpred/constants.py

```diff
@@ -2,6 +2,15 @@
 CLIMPRED_ENSEMBLE_DIMS = ['init', 'member', 'lead']
 CLIMPRED_DIMS = CLIMPRED_ENSEMBLE_DIMS + ['time']
 
+# List of frequencies to check to infer time series stride
+FREQ_LIST_TO_INFER_STRIDE = ['day', 'month', 'year']
+
+# calendar type for perfect-model (PM) (needed for bootstrapping_uninit)
+# Leap also works, but changing Leap,NoLeap fails
+PM_CALENDAR_STR = 'DatetimeNoLeap'
+# standard calendar for hindcast experiments
+HINDCAST_CALENDAR_STR = 'DatetimeProlepticGregorian'
```

Comment:
This will probably change down the road. Most hindcast experiments are also `DatetimeNoLeap`. I just used real calendars for now to compare to the observations, since the observations in some cases have leap years. But the user can bypass this by supplying objects with `cftime` already.

Reply:
Will also `daily` hindcasts have `NoLeap`. I understand this for our decadal experiments. Unsure about shorter timescales. If this ends up being very diverse, we will have to introduce a `.config.yaml` file where the user can set those constants themselves.

Reply:
Yes I think eventually having a `config.yaml` file for the bigger stuff will be good. We can learn from how Luke handles it in `proplot`.